docs/index.adoc

```diff
@@ -14,3 +14,4 @@ include::owaspMobileVerificationStandard.adoc[leveloffset=+1]
 include::androidSecurityGuide.adoc[leveloffset=+1]
 include::nispMobileSecurityVetting.adoc[leveloffset=+1]
 include::nispMobileDeviceSecurityGuides.adoc[leveloffset=+1]
+include::niapMobileAppVetting.adoc[leveloffset=+1]
```

Comment:
I think better put this right after the `nispMobileSecurityVetting.adoc` file as they are kind related.